support/Chart.yaml

```diff
@@ -29,3 +29,9 @@ dependencies:
     version: 9.10.3
     repository: https://kubernetes.github.io/autoscaler
     condition: cluster-autoscaler.enabled
+
+  # In-cluster NFS server provisioner
+  - name: nfs-server-provisioner
+    version: 1.3.1
+    repository: https://kvaps.github.io/charts
+    condition: nfs-server-provisioner.enabled
```

Comment:
Let's turn this off by default?

Reply:
Done in [`8242a88`](https://github.com/2i2c-org/pilot-hubs/pull/613/commits/8242a88778bdc968b8fa0b934ff3dccb32814c94))